packages/litexa-html/lib/htmlHandler.js

```diff
@@ -1,23 +1,57 @@
 module.exports = (context) => {
-    const contextHasHTML = () => {
-        const interfaces =
-            context &&
-            context.event &&
-            context.event.context &&
-            context.event.context.System &&
-            context.event.context.System.device &&
-            context.event.context.System.device.supportedInterfaces;
-        return interfaces && interfaces.hasOwnProperty('Alexa.Presentation.HTML');
-    };
+  const contextHasHTML = () => {
+    const interfaces =
+      context &&
+      context.event &&
+      context.event.context &&
+      context.event.context.System &&
+      context.event.context.System.device &&
+      context.event.context.System.device.supportedInterfaces;
+    return interfaces && interfaces.hasOwnProperty('Alexa.Presentation.HTML');
+  };
+
+  let userFacing = {};
+  let isEnabled = true;
+
+  userFacing.setEnabled = (enabled) => isEnabled = enabled;
+  userFacing.isHTMLPresent = () => contextHasHTML() && isEnabled;
+  
+  userFacing.mark = (str) => {
+    if (!contextHasHTML()) { return; }
+    context.say.push(`<mark name='${str}'/>`);
+  }
 
-    return {
-        userFacing: {
-            isHTMLPresent: () => contextHasHTML(),
-            mark: (str) => {
-                if (!contextHasHTML()) { return; }
-                context.say.push(`<mark name='${str}'/>`);
-            }
-        },
-        events: {}
+  userFacing.start = (url, timeout, initialData, transformers) => {
+    if (!contextHasHTML()) { return null; }
+    timeout = timeout || 480;
+    initialData = initialData || undefined;
+    transformers = transformers || undefined;
+    let absoluteUrl = url;
```

Comment:
Could validate that `url` is a string, to catch it with a more indicative error than the `indexOf` failing.